<launch>
    <!-- use different namespaces to launh with different params (used for modularity)-->
    <group ns="testRobot"> 
        <node name="motor_controller_driver" pkg="robot_driver" type="motor_controller_driver" output="screen">
            <param name="max_speed" type="int" value="5" />
            <param name="publish_speed_frequency" type="double" value="10.0" />
            <param name="publish_status_frequency" type="double" value="2.0" />
        </node>
    </group>
</launch>

